tests/unit/utils/get-header-test.js

```diff
@@ -0,0 +1,44 @@
+/* jshint expr:true */
+import getHeader from 'ember-ajax/utils/get-header';
+import { describe, it } from 'mocha';
+import { assert } from 'chai';
```

Comment:
Would you mind using `expect` instead? The codebase might not be there right now, but I'd like move in that direction.